demo/site/src/blocks/seo/SeoBlock.tsx

```diff
@@ -5,12 +5,10 @@ import * as React from "react";
 
 interface SeoBlockProps extends PropsWithData<SeoBlockData> {
     title: string;
-    canonicalUrl: string;
```

Comment:
I'd still keep it as an option to allow developers to override the canonical URL in the frontend

Reply:
However, the canonical URL specified by the user should be picked first: `data.canonicalUrl ?? props.canonicalUrl`